src/character.cpp

```diff
@@ -4065,6 +4094,9 @@ void Character::set_stored_calories( int cal )
 
         //some mutant change their max_hp according to their bmi
         recalc_hp();
+        //need to check obesity penalties when this happens
+        calc_encumbrance();
```

Comment:
Duplicated and redundant with the calls above. Should probably just be in this function